packages/server/src/api/controllers/static/index.ts

```diff
@@ -167,10 +222,23 @@ export const serveApp = async function (ctx: Ctx) {
     const appInfo = await db.get<any>(DocumentType.APP_METADATA)
     let appId = context.getAppId()
 
+    const hideDevTools = !!ctx.params.appUrl
+    const sideNav = appInfo.navigation.navigation === "Left"
+    const hideFooter = ctx.user?.license?.features?.includes("branding")
+    const themeVariables = getThemeVariables(appInfo?.theme)
+
     if (!env.isJest()) {
       const plugins = objectStore.enrichPluginURLs(appInfo.usedPlugins)
+      const Skeleton =
```

Comment:
Have to manually load the skeleton component here instead of putting it in the base template. 

Our build step seems to copy svelte files to the server dist folder, where they are processed at run time, so import components in components doesn't really work.

They need to be rendered individually like this or manually copied to where they're expected in the build step, and even then the paths will be different than you would expect, so this method is best imo.